### ISSUE TITLE
Outline fails to connect after Windows wakeup from sleep/suspend

### ISSUE THREAD

**Issue by mfat** [Jigsaw-Code/outline-client#1080]
Outline client works perfectly on Windows 10 until I leave my laptop on for an extended period of time. When Windows sleeps/suspends, Outline there is no internet until I quit (not close) outline.

**Comment by ahmed2m** [comment]
Same on Linux

**Comment by daniellacosse** [comment]
Wonder if it's related: https://github.com/Jigsaw-Code/outline-client/issues/1087

**Comment by skliarovartem** [comment]
Same with Mac. Even after close and run back it can't connect to server...

**Comment by skliarovartem** [comment]
Anything helps... it works again only after I restart my mac.

**Comment by skliarovartem** [comment]
Nope, for me easy just restart my laptop. It look like some process is still working when I close outline and I need to close it or restart. Maybe you have ideas what it can be?

**Comment by daniellacosse** [comment]
What's the name of the process?

For context, I'm a fairly new maintainer and am still learning the ropes.

**Comment by skliarovartem** [comment]
"What's the name of the process?" this is what I wanted to know :) what I should to restart to make Outline working again without restarting laptop

**Comment by daniellacosse** [comment]
Oh, I see, I thought maybe you could see the process in activity monitor or something

**Comment by Alphyra** [comment]
Hi! Any updates on this issue?

**Comment by qwaver** [comment]
Same windows 10 pro, after hibernation

https://user-images.githubusercontent.com/44930484/172646947-6f23fafc-ba16-43c6-8580-bd0e58f86a8d.mp4

**Comment by fffelix-jan** [comment]
Same on Windows 11; you need to restart the client to make it work again.

**Comment by hmirheydari** [comment]
After windows 10 wake up while outline shows it is connected but I have not any internet connectivity, only has access to LAN, after disconnecting outline, I have access to internet

**Comment by alexander-potemkin** [comment]
The issue happens now with WebSocket only - not with ShadowSocks. Any chances to check that / fix? Any troubleshooting information that is required?

**Comment by alexander-potemkin** [comment]
Got the error the user is getting from Outline:

```
ProxyConnectionFailure: Failed to connect to server dynamic_url.server.com.
Cause:   ERR_INTERNAL_ERROR: Internal service error
  Cause:     Error: time=2026-02-19T10:45:38.704+03:00 level=INFO msg="TAP device component ID: {F3F1FB13-E519-4AA8-877D-B50593542C18}"
time=2026-02-19T10:45:38.722+03:00 level=INFO msg="TAP device name: outline-tap0"
{"code":"ERR_SYSTEM_VPN_SETUP_FAILURE","message":"failed to open TUN device","cause":{"code":"ERR_INTERNAL_ERROR","message":"A device attached to the system is not functioning."}}
```

@maddyhof , @fortuna , would it be of any help? If not - any other information might be?
